File: fibonacci_sequence.cpp
#include <iostream>
#include <iomanip>
using namespace std;
 
long double fib[1000000]; int x;
 
int main()
{
    cout << "How many Fibonacci numbers: ";
    cin>>x;
 
    fib[0]=0;
    fib[1]=1;
 
    for (int i=2; i<x; i++)
    {
        fib[i] = fib[i-1] + fib[i-2];
    }
cout<<setprecision(1000000);

    for (int i=0; i<x; i++)
    {
        cout<<fib[i]<<"  ";
    }
cout<<endl;

    return 0;
}

File: fibonacci_sequence.py
####fibonacci_sequence

x = input("How many Fibonacci numbers:")
x = int(x)

fib = [0, 1]

i = 2
while i < x:
    fib.append(fib[i-1] + fib[i-2])
    i += 1

print("Fibonacci numbers:", fib[:x])
